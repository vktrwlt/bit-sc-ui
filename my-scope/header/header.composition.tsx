import { Header } from './header';

export const BasicHeader = () => {
  return (
      <Header fontSize={[4, 5]}>Hello World!</Header>
  );
}
