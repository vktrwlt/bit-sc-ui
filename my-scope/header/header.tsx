import styled from 'styled-components';

import { typography, TypographyProps } from 'styled-system';

export type HeaderProps = TypographyProps;

export const Header = styled.div<HeaderProps>`
  ${typography}
`;
