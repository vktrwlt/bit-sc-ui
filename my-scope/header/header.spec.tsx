import React from 'react';
import { render } from '@testing-library/react';
import { BasicHeader } from './header.composition';

it('renders with the correct text', () => {
  const { getByText } = render(<BasicHeader />);
  const rendered = getByText('Hello World!');

  expect(rendered).toBeTruthy();
});

it('matches snapshot', () => {
  const {container} =  render(<BasicHeader />);
  

  expect(container).toMatchSnapshot();
});
