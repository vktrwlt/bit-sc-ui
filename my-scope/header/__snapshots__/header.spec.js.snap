// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
.c0 {
  font-size: 24px;
}

@media screen and (min-width:40em) {
  .c0 {
    font-size: 32px;
  }
}

<div>
  <div
    class="c0"
    font-size="4,5"
  >
    Hello World!
  </div>
</div>
`;
